src/app/app.module.ts

```diff
@@ -29,7 +30,8 @@ import {UserService} from './user.service'
     AppComponent,
     LoginComponent,
     SelectRepositoryComponent,
-    CohortBuilderComponent
+    CohortBuilderComponent,
+    SignInComponent
```

Comment:
I can't get my component to work without this line, but this can't be right. It would be awful to have to import every single component here.